lib/iris/coords.py

```diff
@@ -442,31 +442,36 @@ def __init__(self, points, standard_name=None, long_name=None,
             integer or boolean masked bounds.
 
         """
-        #: CF standard name of the quantity that the coordinate represents.
+        # CF standard name of the quantity that the coordinate represents.
```

Comment:
I think these colons make these comments get added to the docstring for the following attribute, so you might want to keep them